docker-compose.yml

```diff
@@ -38,6 +38,7 @@ services:
       CHAIN_ID: "fetchchain"
     volumes:
       - ./:/app
+      - ./docker/node-cosmos/package.json:/usr/local/lib/node_modules/@subql/node-cosmos/package.json
```

Comment:
This (and :72) look like they're from #226

Reply:
(same for api.dockerfile and node-cosmos/package.json

Reply:
You can use interactive rebase to drop that commit:

```
git rebase -i 7867f22^
```

Then delete the first line (or change its "pick" to "drop" or "d"; see the in-editor instructions for more):
```
pick 7867f22 chore: set version & add package.json volume mounts # <-- this one
pick f07ade7 fix: amend schema and contract creation to include single interface only
pick 31caefb docs: readme: expand on how to use e2e tests (#216)
pick 0ee62f9 fix: rename filter query (#214)
pick 090ce81 feat: add contract to execute msg (#210)
pick 24c9d48 chore: set concrete cosmpy version (#219)
pick db23436 fix: package.json remove duplicate script (#221)
pick 12b824f docs: readme tweak (#222)
pick dcbc45f Linting cl checks (#179)
pick 1cc195f fix: set WARNING level on gql aiohttp transport logger (#223)
pick c7bb74f feat: add contract relation to cw20 transfer (#213)
pick 8ed13fb lint: Added ignoring of E203 (#224)
pick d01bc53 fix: amend schema, contract and tests to include only a single interface
pick b687255 refact: revert unnecessary field name change
```